From 6bce5200673bfc811ac01f09340295fd3f1c8f4f Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Sat, 27 Jun 2020 12:33:18 +0800
Subject: [PATCH 6/8] net: phy: drop unused label out
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

To fix below warning:
drivers/net/phy/phy_device.c:279:1: warning: label 'out' defined but not used [-Wunused-label]
  279 | out:
      | ^~~

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/phy/phy_device.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/phy/phy_device.c b/drivers/net/phy/phy_device.c
index a8f1f0d8a9a1..5f5ee696b23c 100644
--- a/drivers/net/phy/phy_device.c
+++ b/drivers/net/phy/phy_device.c
@@ -276,7 +276,6 @@ static bool mdio_bus_phy_may_suspend(struct phy_device *phydev)
 	if (device_may_wakeup(&netdev->dev))
 		return false;
 
-out:
 	return !phydev->suspended;
 }
 
-- 
2.17.1

